std_execution.bs

```diff
@@ -553,6 +553,17 @@ merged into a single one in an asynchronous fashion.
 `transfer_when_all` accepts an additional scheduler argument. It returns a sender whose value [=completion scheduler=] is the scheduler provided as an argument, but otherwise behaves the same as `when_all`. You can think of it as a composition of
 `transfer(when_all(inputs...), scheduler)`, but one that allows for better efficiency through customization.
 
+## Schedulers advertise their forward progress guarantees ## {#design-fpg}
+
+To decide whether a scheduler (and its associated execution context) is sufficient for a specific task, it may often be necessary to know what kind of forward progress guarantees it provides for the execution agents it creates. The C++ Standard defines the following
+forward progress guarantees:
+
+* <i>concurrent</i>, which requires that a thread makes progress <i>eventually</i>;
+* <i>parallel</i>, which requires that a thread makes progress once it executes a step; and
+* <i>weakly parallel</i>, which does not require that the thread makes progress.
+
+This paper introduces a scheduler query function, `get_forward_progress_guarantee`, which returns one of values of a new `enum` type, `forward_progress_guarantee`, which matches one of the gurantees defined by the C++ Standard.
```

Comment:
"one of values of a new `enum` type" -> "one of the enumerators of a new `enum` type".

Reply:
"`forward_progress_guarantee`, which matches one of the guarantees defined by the C++ Standard." -> "`forward_progress_gaurantee`. Each enumerator of `forward_progress_guarantee` corresponds to one of the aforementioned guarantees."

(Split it into two sentences, and reword the second one)